dockerfiles/jupyter/Dockerfile

```diff
@@ -27,6 +27,16 @@ RUN cd /tmp/mlrun && python -m pip install ".[complete-api]" && cd /tmp && rm -r
 
 RUN chown -R $NB_UID:$NB_GID $HOME
 
+RUN apt-get update && apt-get --yes install \
+    apt-utils \
+    curl
+# install whatever else you want on this line
```

Comment:
Remove what? I kept the `curl` since it was there previously and is needed to retrieve `kubectl` later on. I think this was here before I did my changes, not sure why it appears that I've added it (maybe due to the various changes I did).

Reply:
And the comment was there before - I didn't add it...

Reply:
Mmm I don't think github diff is lying, I meant to the comment, please remove it

Reply:
yeah, I can't remember where it came from. Probably copied it from somewhere. Will remove the entire block.